public/data/devmukhtarr.json

```
@@ -24,5 +24,22 @@
       "url": "https://www.linkedin.com/in/abdulfatai-mukhtar-77a2131b1/",
       "icon": "linkedIn"
     }
+  ],
+  "milestones": [
+    {
+      "title": "Wrote my first blog post on hashnode",
+      "date": "Feb 6, 2021",
+      "icon": "hashnode",
+      "color": "blue",
+      "description": "Wrote my first ever blog post which was about setting up nodejs and published it on hashnode",
```

To make it look more professional, what do you think if we capitalize the following words?
```suggestion
      "description": "Wrote my first ever blog post which was about setting up NodeJS and published it on Hashnode",
```